input/drivers/udev_input.c

```diff
@@ -4215,13 +4336,54 @@ static void udev_input_grab_mouse(void *data, bool state)
 #endif
 }
 
+
+static float udev_input_get_sensor_input(void *data, unsigned port, unsigned id) {
+   const udev_input_t *udev = (const udev_input_t *)data;
+   const udev_input_sensor_t * sensor;
+   float sensor_value;
+   udev_input_sensor_limits_t limits;
+
+   if (!udev || udev->devices == NULL) return 0.f;
+
+   sensor=udev_get_sensor(udev,port);
+   if (!sensor) return 0.f;
+   if (id >= RETRO_SENSOR_ACCELEROMETER_X && id <= RETRO_SENSOR_GYROSCOPE_Z){
+      sensor_value=(float)sensor->sensor_data.a[id-RETRO_SENSOR_ACCELEROMETER_X];
+      limits=sensor->limits.a[id-RETRO_SENSOR_ACCELEROMETER_X];
+   }
+   else
+      return 0.f;
+   
+   if (id == RETRO_SENSOR_ILLUMINANCE && udev->illuminance_sensor)
+      return linux_get_illuminance_reading(udev->illuminance_sensor);
+   
+   RARCH_DBG(
+      "[udev] sensor:\n"
+      "\t%d\n"
+      "\t%d\n"
+      "\t%f\n",
+      id,
+      port,
+      sensor_value
+   );
```

Comment:
Don't forget to remove this when you're done, or else it'll spam the logs.

Reply:
IDK it will be probably quite useful to debug the odd "Hey sensor not workig halp". Though there's probably a better way.

Reply:
Wouldn't that be indicated with an error code, or a failure to open the device in the first place? I can understand using logging as a quick debugging aide, but leaving this in as-is _will_ make it more difficult to troubleshoot other issues.